fidesctl/src/fidesctl/cli/commands/util_comands.py

```diff
@@ -0,0 +1,133 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
+    """
+    Initialize a Fidesctl instance.
+    """
+
+    # Constants
+    dir_name = ".fides"
+    config_path = f"{dir_name}/fidesctl.toml"
+    config = ctx.obj["CONFIG"]
+
+    # List the values we want to exclude from the user-facing config
+    # These are values that are generated at config load time
+    excluded_values = {
+        "user": {"request_headers"},
+        "api": {"async_database_url", "sync_database_url"},
+    }
+
+    echo_green("Initializing Fidesctl...\n")
```

Comment:
This doesn't indicate that anything was successful.
```suggestion
    print("Initializing Fidesctl...\n")
```

Reply:
done